docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -143,284 +134,212 @@ Please set the important parameters in iotdb-datanode.properties:
 
-Remove on Linux:
-```
-# Remove the ConfigNode with confignode_id
-./confignode/sbin/remove-confignode.sh <confignode_id>
+| **Configuration**                   | **Description**                                                                                                                                            |
+|-------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| dn\_rpc\_address                    | Client RPC Service address                                                                                                                                 |
+| dn\_rpc\_port                       | Client RPC Service port                                                                                                                                    |
+| dn\_internal\_address               | Control flow address of DataNode inside cluster                                                                                                            |
+| dn\_internal\_port                  | Control flow port of DataNode inside cluster                                                                                                               |
+| dn\_mpp\_data\_exchange\_port       | Data flow port of DataNode inside cluster                                                                                                                  |
+| dn\_data\_region\_consensus\_port   | Data replicas communication port for consensus                                                                                                             |
+| dn\_schema\_region\_consensus\_port | Schema replicas communication port for consensus                                                                                                           |
+| dn\_target\_config\_node\_list      | Configure the internal communication address of an arbitrary running ConfigNode. The internal communication address of the seed ConfigNode is recommended. |
 
-# Remove the ConfigNode with address:port
-./confignode/sbin/remove-confignode.sh <internal_address>:<internal_port>
+After the configuration is complete, run the startup script:
 ```
+# Linux foreground
+bash ./sbin/start-datanode.sh
 
-Remove on Windows:
-```
-# Remove the ConfigNode with confignode_id
-confignode\sbin\remove-confignode.bat <confignode_id>
+# Linux background
+nohup bash ./sbin/start-datanode.sh >/dev/null 2>&1 &
 
-# Remove the ConfigNode with address:port
-confignode\sbin\remove-confignode.bat <internal_address>:<internal_port>
+# Windows
+.\sbin\start-datanode.bat
 ```
 
-### Remove DataNode
+For more details about other configuration parameters of DataNode, see the
+[DataNode Configurations](https://iotdb.apache.org/UserGuide/Master/Reference/DataNode-Config-Manual.html).
 
```

Comment:
Add a chapter to check whether the cluster is deployed successfully.

Reply:
Added in chap 6.3